tests/tests/test-balance/test-balance-transfer.ts

```diff
@@ -104,6 +104,7 @@ describeDevMoonbeam(
         createTransaction(context, {
           ...ALITH_TRANSACTION_TEMPLATE,
           gas: "0x5208",
+          gasPrice: undefined, // clear legacy field and use EIP-1559 instead
```

Comment:
This feels clunky, it's a consequence of burying `gasPrice` inside `TRANSACTION_TEMPLATE`. Maybe `gasPrice` could be omitted there since `createTransaction` will infer it... :thinking:

Reply:
I was able to remove it in 18c7565. Next step: can we use `web3.eth.getGasPrice()` for `gasPrice` when not provided?